apps/src/blockly/addons/contextMenuAndShortcut.js

```diff
@@ -219,4 +168,5 @@ function hasShadowChildren(block) {
   return shadowChildCount(block) > 0;
 }
 
-exports.registerAllContextMenuItems = registerAllContextMenuItems;
+// shortcuts are registered by the CrossTabCopyPaste plugin
+exports.registerAllContextMenuShortcutItems = registerAllContextMenuItems;
```

Comment:
`registerAllContextMenuItems` still only includes context menu options (and no shortcuts). `registerAllContextMenuShortcutItems` feels over-complex and a little inaccurate give there aren't any shortcuts being exported.

Reply:
That definitely makes sense -just pushed the changes - thanks!